app/src/main/java/de/westnordost/streetcomplete/quests/way_lit/AddWayLit.java

```diff
@@ -13,12 +13,14 @@
 
 public class AddWayLit extends SimpleOverpassQuestType
 {
+	static final String[] LIT_RESIDENTIAL_ROADS = { "residential", "living_street" };
+
 	static final String[] LIT_ROADS = {
 			"primary", "secondary", "tertiary", "unclassified", "residential", "living_street",
 			"service", "pedestrian"
```

Comment:
Since you you use these separately, you should rename it to `LIT_NON_RESIDENTIAL_ROADS` and remove residential, living_street and pedestrian.
(Oh yeah, pedestrian would also be residential)